src/pre_defined_classes/mathematical_program/hard/q9/MainTest.java

```diff
@@ -0,0 +1,27 @@
+package pre_defined_classes.mathematical_program.hard.q9;
+
+import global.BaseTest;
+import global.variables.Clause;
+import global.variables.clauses.*;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class MainTest extends BaseTest {
+    // Parsons (with distractors)
+    public Clause[] testSentence() {
+        return new Clause[]{
+                new StringLiteral("The growth rate for the country is:"),
+                new DoubleLiteral("growthRate")
+        };
+    }
+
+    public void runMain() {
+        GrowthRate.main(new String[]{});
+    }
+
+    @Test
+    public void mathTest() {
+        assertEquals(56.25, Double.parseDouble(getItemByName("growthRate")), 0.1, "Math is incorrect. Refer back to the question for what the correct calculation should look like");
```

Comment:
I would say something more along the lines of growthRate is incorrect, but I think that is a personal preference.